utility/PlexPreferNonForcedSubs.py

```diff
@@ -0,0 +1,64 @@
+from plexapi.server import PlexServer
+import requests
+import xml.etree.ElementTree as ET
+#import pdb             //enable for debugging
+#pdb.set_trace()       //insert where to start debugging, press n to proceed to next line
+
+#This python script will set all movies and shows in your local Plex library to English non forced subtitles by default. The subtitle selections will apply to your Plex profile and be remembered on other devices.
+
+# Connect to the Plex server
+baseurl = 'http://localhost:32400'
+token = 'xxxxxxxx'
+plex = PlexServer(baseurl, token)
+
+
+# Set all movies to use English non-forced subtitles if available, otherwise print no subtitles found
+for movie in plex.library.section('Movies').all():

```

Comment:
You're assuming the library name. Use a variable, arg, or media type to select the proper library.